contracts/interfaces/IPeripheryPayments.sol

```diff
@@ -10,7 +10,7 @@ interface IPeripheryPayments {
     /// @param recipient The address receiving ETH
     function unwrapWETH9(uint256 amountMinimum, address recipient) external payable;
 
-    /// @notice Sends the full amount of a token held by this contract to the given recipient
+    /// @notice Calculates the full amount of a token held by this contract and sends it to the given recipient
```

Comment:
idk about calculates, i don't mind the other description, but since this is changing, maybe `Transfers` is more specific?